R/plotEGRET.R

```diff
@@ -0,0 +1,30 @@
+#' Create an EGRET-style plot
+#'  
+#' Use a loadflex object, but create an EGRET plot.
+#' 
+#' @param intdat data.frame of interpolation data
+#' @param estdat data.frame of estimation data
+#' @param preds data.frame of load predictions
+#' @param meta loadflex metadata object; it must include constituent,
+#' flow, conc.units, custom (station abbreviation: staAbbr)
+#' @param plotName the name of the plot the user wants to create. Current
+#' options: ConcTime, ConcQ, FluxTimeDaily, and FluxQ.
+#' @param ... additional arguments to pass to the plot
+#'
+#' @importFrom EGRET plotConcTime
+#' @importFrom EGRET plotConcQ
+#' @importFrom EGRET plotFluxTimeDaily
+#' @importFrom EGRET plotFluxQ
+#'
+#' @export
+plotEGRET <- function(intdat, estdat, preds, meta, plotName, ...){
+  
+  egretobj <- convertToEGRET(intdat, estdat, preds, meta)
+  
+  switch(plotName,
```

Comment:
I use the method of first value = default a whole lot, but you might be right that most folks won't find that intuitive (and maybe I should rethink my usual approach). So OK, how about never mind 1 and 2, just do 3 = use the final argument to `switch` to throw an error? (I forget whether you're actually supposed to name it `default`...maybe just leave it unnamed? test it out)

I like the multi-plot option as a long-term feature. Can we hold off for now to save dev time?